configure

```diff
@@ -97,6 +98,7 @@ Options [defaults in brackets]:
   --srcdir=SRCDIR             source code path [$srcdir]
   --with-gcc=DIR              use gcc backend from DIR
   --with-llvm-config=PATH     use llvm from PATH (needs llvm 3.5 - 3.9)
+  --with-gnatmake=PATH        use gnatmake from PATH
```

Comment:
You can already override gnatmake with: xx/configure ... GNATMAKE=my-gnatmake
So, no need for this new option. (But doc improvement is fine!)

Contrary to llvm or gcc, gnatmake is only used to build ghdl

Reply:
Thank you for the comment. The idea with the -- option was was to have a similar user interface for specifying external tools. The option has now been removed from the pull request.